flutter/test/integrations/native_app_start_integration_test.dart

```diff
@@ -207,7 +263,7 @@ void main() {
       final processor = fixture.options.eventProcessors.first;
       tracer = fixture.createTracer();
       final transaction = SentryTransaction(tracer);
-      final enriched =
+      enriched =
```

Comment:
is the removal of `final` an overlook?

Reply:
I set `late SentryTransaction enriched;` so I can access it in each tests